layouts/_default/baseof.html

```diff
@@ -37,7 +37,10 @@
     {{ .Hugo.Generator }}
   </head>
 
-  <body class="{{ if .Site.Params.rtl }}rtl{{ end }}">
+  <body
+    {{ if .Site.Params.invertedColors }} class="inverted-colors" {{ end }}
+    {{ if .Site.Params.rtl }} class="rtl" {{ end }}
```

Comment:
This way, if we use both `invertedColors` and `rtl`, the result will be a double declaration of `class=`.
I don't know how to solve this problem yet. I'll look at it later and tell here.

Reply:
What about this?
```html
<body class="{{ if .Site.Params.invertedColors }}inverted-colors{{ end }} {{ if .Site.Params.rtl }} rtl{{ end }}">```

Reply:
If both options are false, they will leave and empty class attributre. `class=""`
Not cool too.

Reply:
It is exactly like that currently though. Does that have other than cosmetic impact? 

We can do more branching, but it would become quite polluted imo.

Reply:
You can leave this part as you suggested in the comment above.

If we come up with a better option here, we can change it later.